### PR TITLE
Add some tests for <noscript> in scripting disabled case

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

File: tree-construction/noscript01.dat [hunk 1]
```diff
@@ -0,0 +1,235 @@
+#data
+<head><noscript><!doctype html></noscript>
```
Comment (zcorpan) [inline tree-construction/noscript01.dat]: Add a comment after the doctype to see that it ends up inside the noscript

File: tree-construction/noscript01.dat [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+Line: 1 Col: 6 Unexpected start tag (head). Expected DOCTYPE.
+Line: 1 Col: 31 Unexpected DOCTYPE. Ignored.
+#document
+| <html>
+|   <head>
+|     <noscript>
+|   <body>
+
+#data
+<head><noscript><html class="foo"></noscript>
```
Comment (zcorpan) [inline tree-construction/noscript01.dat]: Add a comment after the html tag to see that it ends up inside the noscript

**Comment by zcorpan** [comment]
LGTM % 2 comments

**Comment by gsnedders** [comment]
@zcorpan LGTY?

**Pull request merged**